examples/distributed/model_a.ts

```diff
@@ -21,4 +21,4 @@ class MLP extends sm.module.Module {
 
 const model = new MLP()
 
-sm.network.serve_model(model, (ts) => sm.optim.sgd(ts, 1e-4), { port: 3001 })
+sm.network.serve_model(model.forward, (ts) => sm.optim.sgd(ts, 1e-4), { port: 3001 })
```

Comment:
this should be fine, is it a type error?

Reply:
Yea, it was a type error; `serve_model` expects the first arg to be `(t: sm.Tensor) => sm.Tensor | Promise<sm.Tensor>`

Reply:
I can update the argument type (and make any needed changes in the logic) in `serve_model` though if that's the route to go here